src/utils/helpers/getDateDifference.js

```diff
@@ -0,0 +1,7 @@
+export const getDateDifference = (from) => {
+  let date = new Date(from);
```

Comment:
can't we use moment here? then this calculation won't be needed. There are already methods available in moment

Reply:
I don't see using moment here